stdlib/contrib/jsternberg/influxdb/influxdb.flux

```diff
@@ -1,18 +1,38 @@
+// Package influxdb provides additional functions for querying data from InfluxDB.
+//
+// introduced: 0.77.0
 package influxdb
 
 
 import "influxdata/influxdb"
 import "influxdata/influxdb/v1"
 
+// _mask does TODO
```

Comment:
A change was committed that values prefixed with `_` do not need to be documented. I am still happy to see docs for these functions. Should we just remove the doc comment entirely instead of leaving a TODO?